pennylane/hf/tapering.py

```diff
@@ -464,3 +465,65 @@ def transform_hamiltonian(h, generators, paulix_ops, paulix_sector):
     c = qml.math.stack(c)
 
     return _simplify(qml.Hamiltonian(c, o))
+
+
+def optimal_sector(qubit_op, generators, active_electrons):
+    r"""Get the optimal sector which contains the ground state.
+
+    To obtain the optimal sector, we need to choose the right eigenvalue for the symmetry generators :math:`\bm{\tau}`.
+    We can do so by using the following relation between the Pauli-Z qubit operator and the occupation number under a
+    Jordan-Wigner transform.
+
+    .. math::
+
+        \sigma_{i}^{z} = I - 2a_{i}^{\dagger}a_{i}
+
+    According to this relation, an occupied and unoccupied fermionic mode corresponds to the -1 and +1 eigenvalue of
+    the Pauli-Z operator, respectively. Since all :math:`\bm{\tau}` consist only of :math:`I` and Pauli-Z operators,
```

Comment:
```suggestion
    the Pauli-Z operator, respectively. Since all of the generators :math:`\bm{\tau}` consist only of :math:`I` and Pauli-Z operators,
```